plugins/chat/app/services/chat/start_reply.rb

```diff
@@ -18,33 +18,32 @@ class StartReply
     #   @return [Service::Base::Context]
     params do
       attribute :channel_id, :integer
+      attribute :thread_id, :integer
+
       validates :channel_id, presence: true
 
-      attribute :thread_id, :integer
+      def channel_name
```

Comment:
It’s probably something we completely underuse, but it makes a lot of sense to let the contract manipulates its own data.